src/octokit_utils/index.ts

```diff
@@ -1,127 +1,181 @@
-import { Context, Octokit } from 'probot' // eslint-disable-line no-unused-vars
-import Webhooks from 'probot/node_modules/@octokit/webhooks' // eslint-disable-line no-unused-vars
-import { UserInfo, ReviewLookupResult } from '../types' // eslint-disable-line no-unused-vars
-import { getOwnershipRules } from '../config_parser'
-import { ownsAllFiles } from '../rule_matcher'
-import { composeReviewDismissalMsg } from '../msg_composer'
-import { APP_CHECK_NAME } from '../config'
+import { Context, Octokit } from "probot"; // eslint-disable-line no-unused-vars
+import Webhooks from "probot/node_modules/@octokit/webhooks"; // eslint-disable-line no-unused-vars
+import { UserInfo, ReviewLookupResult } from "../types"; // eslint-disable-line no-unused-vars
+import { getOwnershipRules } from "../config_parser";
+import { ownsAllFiles } from "../rule_matcher";
+import { composeReviewDismissalMsg } from "../msg_composer";
+import { APP_CHECK_NAME } from "../config";
 
-function getPullAuthor(context: Context<Webhooks.WebhookPayloadPullRequest>): string {
-  return context.payload.pull_request.user.login
+function getPullAuthor(
+  context: Context<Webhooks.WebhookPayloadPullRequest>
+): string {
+  return context.payload.pull_request.user.login;
 }
 
-function getUserInfo(context: Context<Webhooks.WebhookPayloadPullRequest>): UserInfo {
+function getUserInfo(
+  context: Context<Webhooks.WebhookPayloadPullRequest>
+): UserInfo {
   const info: UserInfo = {
-    username: getPullAuthor(context)
-  }
-  return info
+    username: getPullAuthor(context),
+  };
+  return info;
 }
 
-function initPullRelatedRequest(context: Context<Webhooks.WebhookPayloadPullRequest>): any {
-  const pullNumber = context.payload.pull_request.number
-  const repo = context.payload.repository.name
-  const owner = context.payload.repository.owner.login
-  context.log.info(`Initializing pull related request with ${owner}/${repo} #${pullNumber}`)
-  return { pull_number: pullNumber, owner, repo }
+function initPullRelatedRequest(
+  context: Context<Webhooks.WebhookPayloadPullRequest>
+): any {
+  const pullNumber = context.payload.pull_request.number;
+  const repo = context.payload.repository.name;
+  const owner = context.payload.repository.owner.login;
+  context.log.info(
+    `Initializing pull related request with ${owner}/${repo} #${pullNumber}`
+  );
+  return { pull_number: pullNumber, owner, repo };
```

Comment:
Codacy found an issue: [Identifier 'pull_number' is not in camel case.](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6083797)